clang/include/clang/Basic/DiagnosticDriverKinds.td

```diff
@@ -79,7 +79,7 @@ def warn_drv_unknown_cuda_version: Warning<
   InGroup<CudaUnknownVersion>;
 def err_drv_cuda_host_arch : Error<"unsupported architecture '%0' for host compilation.">;
 def err_drv_no_sycl_libspirv : Error<
-  "cannot find `libspirv-nvptx64--nvidiacl.bc`. Provide path to libspirv library via "
+  "cannot find suitable `libspirv-nvptx64--nvidiacl.bc` variant. Provide path to libspirv library via "
   "-fsycl-libspirv-path, or pass -fno-sycl-libspirv to build without linking with libspirv.">;
```

Comment:
```suggestion
  "cannot find suitable 'libspirv-nvptx64--nvidiacl.bc' variant. Provide path to libspirv library via "
  "'-fsycl-libspirv-path', or pass '-fno-sycl-libspirv' to build without linking with libspirv.">;
```
Since we're in the area touching this, we might as well correct the quoting. Btw, is the double dash expected in the middle of `libspirv-nvptx64--nvidiacl.bc`? I am guessing yes, but it looks funny so I figured I'd call it out.

Reply:
> Since we're in the area touching this, we might as well correct the quoting.

Sure! 👍

> Btw, is the double dash expected in the middle of libspirv-nvptx64--nvidiacl.bc? I am guessing yes, but it looks funny so I figured I'd call it out.

The double  dash is indeed expected. For this respective example it is the libclc libspirv bitcode file for the `nvptx64--nvidiacl` triple. To my knowledge, this is just a way of specifying that the vendor is unknown. There are also versions without the "OS" as well, eg. there's a version using the `nvptx64--` triple, for which the libspirv bitcode file is `libspirv-nvptx64--.bc` but that is a bit besides the point.

I think in other places we've used "unknown" instead of an empty string, but since libclc uses this naming in upstream LLVM and these libclc projects are likely already in use elsewhere, I don't think it's worth the trouble changing it now.

Reply:
Oh! I learned something new today! Thanks for confirming for me. :-)